benchmarks/graphics_pipeline/GraphicsBenchmarkApp.h

```diff
@@ -32,9 +33,40 @@ static constexpr uint32_t kMaxSphereInstanceCount  = 3000;
 static constexpr uint32_t kSeed                    = 89977;
 static constexpr uint32_t kMaxFullscreenQuadsCount = 1000;
 
+static constexpr float4   kDefaultDrawCallColor        = float4(1.0f, 0.175f, 0.365f, 0.5f);
+static constexpr uint32_t kDebugColorPushConstantCount = sizeof(float4) / sizeof(uint32_t);
+
 static constexpr const char* kShaderBaseDir   = "benchmarks/shaders";
 static constexpr const char* kQuadTextureFile = "benchmarks/textures/resolution.jpg";
 
+enum class DebugView
+{
+    DISABLED,
+    SHOW_DRAWCALLS
+};
+
+inline const int ToInt(DebugView dv)
+{
+    switch (dv) {
+        case DebugView::DISABLED: return 0;
+        case DebugView::SHOW_DRAWCALLS: return 1;
+        default: return -1;
+    }
+}
```

Comment:
```suggestion
enum class DebugView
{
    DISABLED = 0,
    SHOW_DRAWCALLS
};
```

Reply:
all subsequent elements values are last-element + 1. Then, at usage, `(int)DebugView::SHOW_DRAWCALLS`

Reply:
Looks like a simpler way to do it. Thanks. Updated.